lib/mixins/form_state_mixin.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+/// Add this mixin to a [Widget] or [State] to ease use of [Form].
+mixin FormStateMixin<T extends StatefulWidget> on State<T> {
+  /// This Key must be bound with the [Form] Widget that is to be used.
+  final formKey = GlobalKey<FormState>();
+  AutovalidateMode autovalidateMode = AutovalidateMode.disabled;
```

Comment:
remove explicit datatype declaration, use `var` since you are initializing the variable

Reply:
Every thing else is fine.